### DISCUSSION TITLE
How do I remove the background grid dots?

### DISCUSSION THREAD

**Discussion by AliceSum** [xyflow/xyflow#4657] [Q&A]
How do I remove the background grid dots?

**Comment by bcakmakoglu** [comment]
The `<Background>` component can be either lines, dots or cross-shaped, if none of that is to your liking just don't use the component or what exactly is the problem here? 😄

**Comment by AliceSum** [comment]
So if I remove it, the grid dot will be gone? If so, thx for answering.
